cartoframes/geojson.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+
+import base64
+import numpy as np
+
+try:
+    import geopandas
+    HAS_GEOPANDAS = True
+except ImportError:
+    HAS_GEOPANDAS = False
+
+
+def load_geojson(geojson):
+    if not HAS_GEOPANDAS:
+        raise ValueError(
+            """
+            GeoJSON source only works with GeoDataFrames from
+            the geopandas package http://geopandas.org/data_structures.html#geodataframe
+            """)
+
+    if isinstance(geojson, str):
+        data = geopandas.read_file(geojson)
+    elif isinstance(geojson, geopandas.GeoDataFrame):
+        data = geopandas.GeoDataFrame.from_features(geojson)
+    else:
+        raise ValueError(
+            """
+            GeoJSON source only works with GeoDataFrames from
+            the geopandas package http://geopandas.org/data_structures.html#geodataframe
+            """)
+
+    return data
+
+
+def get_query_and_bounds(data):
+    filtered_geometries = _filter_null_geometries(data)
+    bounds = filtered_geometries.total_bounds.tolist()
+    data = _set_time_cols_epoc(filtered_geometries).to_json()
+    query = base64.b64encode(data.encode('utf-8')).decode('utf-8')
```

Comment:
`query` is the base64-encoded geojson? I understand that it will be used in the `query` key for setting up a layer, but at least for this method maybe we can be more transparent about what it is? `encoded_data` or something? Maybe I'm missing something

Reply:
Should this also be an internal function, beginning with a `_`?

Reply:
Yes. I think this was a copy-paste issue. 

It is a public function since it is used inside the Source class, but we could refactor it and move it there.